tools/db/README.md

```diff
@@ -0,0 +1,121 @@
+ 
+
+##### Using an ephemeral CouchDB container
+
+To try out OpenWhisk without managing your own CouchDB installation, you can start a CouchDB instance in a container as part of the OpenWhisk deployment. We advise that you use this method only as a temporary measure. Please note that:
+
+  * no data will persist between two creations of the container
+  * you will need to run the creation script every time you `clean` or `teardown` the system (see below)
+  * you will need to initialize the datastore each time (`tools/db/createImmportalDBs.sh`, see below)
+
+```
+  # Work out of your openwhisk directory
+  cd /your/path/to/openwhisk
+
+  # Start a CouchDB container and create an admin account
+  tools/db/couchdb/start-couchdb-box.sh whisk_admin some_passw0rd
+
+  # The script above automatically creates couchdb-local.env
+  cat couchdb-local.env
+```
+
+#### Using Cloudant
+
+As an alternative to a self-managed CouchDB, you may want to try [Cloudant](https://cloudant.com) which is a cloud-based database service. 
+There are two ways to get a Cloudant account and configure OpenWhisk to use it. 
+You only need to establish an account once, either through IBM Bluemix or with Cloudant directly. 
+
+##### Create a Cloudant account via IBM Bluemix
+Sign up for an account via [IBM Bluemix](https://bluemix.net). Bluemix offers trial accounts and its signup process is straightforward so it is not described here in detail. Using Bluemix, the most convenient way to create a Cloudant instance is via the `cf` command-line tool. See [here](https://www.ng.bluemix.net/docs/starters/install_cli.html) for instructions on how to download and configure `cf` to work with your Bluemix account.
+
+When `cf` is set up, issue the following commands to create a Cloudant database.
+
+  ```
+  # Create a Cloudant service
+  cf create-service cloudantNoSQLDB Shared cloudant-for-openwhisk
+
+  # Create Cloudant service keys
+  cf create-service-key cloudant-for-openwhisk openwhisk
+
+  # Get Cloudant username and password
+  cf service-key cloudant-for-openwhisk openwhisk
+  ```
+
+Make note of the Cloudant `username` and `password` from the last `cf` command so you can create the required `cloudant-local.env`.
+
+##### Create a Cloudant account directly with Cloudant
+
+As an alternative to IBM Bluemix, you may sign up for an account with [Cloudant](https://cloudant.com) directly. Cloudant is free to try and offers a metered pricing where the first $50 of usage is free each month. The signup process is straightforward so it is not described here in detail.
+Once you have created a Cloudant account, make note of the account `username` and `password` from the Cloudant dashboard, so you can create the required `cloudant-local.env`.
+
+##### Setting the Cloudant credentials 
+ 
+Then within your `openwhisk` directory, copy the file `template-cloudant-local.env` to `cloudant-local.env` and edit as appropriate.
```

Comment:
fixed in https://github.com/openwhisk/openwhisk/pull/360/commits/6ce38d87c034e42a9acea3d26bb1a6776f12e739